src/macros.rs

```diff
@@ -69,16 +69,16 @@ macro_rules! to_hex_string {
 #[doc(hidden)]
 #[macro_export]
 macro_rules! core_uri {
```

Comment:
nitpick: when I first saw the `core_uri` I thought it'd be something more "core" to Rust URI representations 😬 But this is related to our `transaction_service`. I'd say that renaming this macro to `transaction_service_uri` would make it more explicit (out of scope for this PR).

Reply:
That's a good point! I took that from a naming suggestion from @rmeissner , but I am open to discus this if it brings more clarity to the code base.